code/__DEFINES/reactions.dm

```diff
@@ -33,7 +33,7 @@
 #define NOBLIUM_RESEARCH_AMOUNT				1000
 #define BZ_RESEARCH_SCALE					4
 #define BZ_RESEARCH_MAX_AMOUNT				400
-#define MIASMA_RESEARCH_AMOUNT				160
+#define MIASMA_RESEARCH_AMOUNT				40
```

Comment:
?????

Reply:
Was changed as a result of the corpse flower being added in 44322.